specification/informatica/Informatica.DataManagement.Management/main.tsp

```diff
@@ -217,6 +235,10 @@ model InfaServerlessFetchConfigProperties {
   @doc("Supplementary File location")
   supplementaryFileLocation?: string;
 
+  @doc("Serverless runtime data disks")
+  @extension("x-ms-identifiers", [])
```

Comment:
>[] [](http://example.com/codeflow?start=33&length=2)

Consider adding some property which will represent unique value in the array, if it is there. [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-identifiers](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-identifiers)